asn1/CommonLedgerProtocol.asn

```diff
@@ -21,81 +21,53 @@ SideProtocolData ::= SEQUENCE OF SEQUENCE {
   protocolData OCTET STRING
 }
 
-PacketRequest ::= SEQUENCE {
-  packet InterledgerPacket,
+Ack ::= SEQUENCE {
+  ack NULL,
   -- Additional data for protocol extensibility
   sideProtocolData SideProtocolData
 }
 
-PacketResponse ::= SEQUENCE {
+Response ::= SEQUENCE {
   packet InterledgerPacket,
```

Comment:
I fixed a bunch of asn bugs including this one in https://github.com/interledger/rfcs/pull/267/files